targettests/execution/uccsd.cpp

```diff
@@ -8,7 +8,7 @@
 
 // clang-format off
 // RUN: nvq++ %cpp_std --target anyon --emulate %s -o %t && %t | FileCheck %s
-// RUN: nvq++ %cpp_std --target braket --emulate %s -o %t && %t | FileCheck %s
+// XUN: if %braket_avail; then nvq++ %cpp_std --target braket --emulate %s -o %t && %t | FileCheck %s ; fi
```

Comment:
It this test failing?

Reply:
Not sure. There is some tensornet test failing.